app/mailers/citizen_mailer.rb

```diff
@@ -1,11 +1,15 @@
 class CitizenMailer < ApplicationMailer
-  def notify_new_citizen(citizen)
-    @citizen = citizen.decorate
+  def notify_new_citizen(citizen_id)
+    @citizen = Citizen.find(citizen_id)
+    return unless @citizen.present?
```

Comment:
Style (opcional): return if @citizen.blank?

Em termos mentais rápidos, segundo os rails best practices, a maioria dos programmers entende mais rápido o if do que o unless (not if). Por isso, quando POSSÍVEL (nem sempre é, por isso o unless é vantajoso tbm) prefira o if.

Reply:
Isso é verdade, eu tbm me confundia antes, já alterei aqui para o blank.